pom.xml

```diff
@@ -28,30 +28,25 @@
     </dependencies>
 
     <build>
-    <plugins>
-    <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <configuration>
-        <archive>
-            <manifest>
-                <mainClass>fully.qualified.MainClass</mainClass>
-            </manifest> 
-        </archive> 
-        <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-        </descriptorRefs>
-        </configuration>
-        <executions>
-            <execution>
-                <id>make-assembly</id>
-                <phase>package</phase>
-                <goals>
-                    <goal>single</goal>
-                </goals>
-            </execution>
-        </executions>
-    </plugin>
-    </plugins>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <configuration>
+                    <descriptorRefs>
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                    </descriptorRefs>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>make-assembly</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
     </build>
 
```

Comment:
skriver du denna ur huvudet eller letade du upp den online? satt och testade en massa olika varitanter av denna i mitt försök att lägga in logging4j i veckan

Reply:
Snodde från nätet... har du gjort det någon gång så vet man ju.
build-plugin gör så att det bundlar de dependencies man har med. Annars måste man ordna åtkomst på annat sätt. Får runtime fel om man inte har allt som det är beroende av.